### PR TITLE
Update library to accommodate open/close states for panels and remove Flex Library references

### PR DESCRIPTION
**Pull request by johnnyraper**

### CODE REVIEW

File: projects/ngx-row-accordion/src/lib/ngx-row-accordion.component.ts [hunk 1]
```diff
@@ -10,6 +10,8 @@ import { NgxRowAccordionService } from './ngx-row-accordion.service';
   styleUrls: ['./ngx-row-accordion.component.scss'],
 })
 export class NgxRowAccordionComponent implements OnInit, OnDestroy {
+
+  constructor(private ngxRowAccordionService: NgxRowAccordionService) {}
```
Comment (maxime1992) [inline projects/ngx-row-accordion/src/lib/ngx-row-accordion.component.ts]: please move the constructor between the properties and the methods.
(line 27)

File: projects/ngx-row-accordion/src/lib/ngx-row-accordion.component.ts [hunk 2]
```diff
@@ -21,7 +23,7 @@ export class NgxRowAccordionComponent implements OnInit, OnDestroy {
   private id: string = uuid();
   private onDestroy$: Subject<void> = new Subject();
 
-  constructor(private ngxRowAccordionService: NgxRowAccordionService) {}
+  public status = false;
```
Comment (maxime1992) [inline projects/ngx-row-accordion/src/lib/ngx-row-accordion.component.ts]: `status` is IMO a bit too generic. Could you rename to something more specific?
Maybe `isPanelOpened`?

File: projects/ngx-row-accordion/src/lib/ngx-row-accordion.component.ts [hunk 3]
```diff
@@ -41,5 +43,6 @@ export class NgxRowAccordionComponent implements OnInit, OnDestroy {
 
   toggle() {
     this.ngxRowAccordionService.toggle(this.id);
+    this.status = !this.status;
```
Comment (maxime1992) [inline projects/ngx-row-accordion/src/lib/ngx-row-accordion.component.ts]: The status should not be defined from there.
It should be contained into the service and the view should be driven by the value coming from the service. This should be an observable.

**Review by maxime1992 — CHANGES_REQUESTED** [review]
Thanks Johnny, it's looking good and removing the need for flex-layout is :fire:. You've also forgotten to remove that dependency btw

**Review by maxime1992 — APPROVED** [review]

**Comment by johnnyraper** [comment]
Thanks for your help on this Maxime, great to add some extra functionality and define the flex properties in the CSS! 😄

**Pull request merged**